lib/membrane/video_compositor/queue/offline/element.ex

```diff
@@ -119,8 +119,11 @@ defmodule Membrane.VideoCompositor.Queue.Offline.Element do
   def handle_parent_notification(
         {:update_scene, scene = %Scene{}},
         _ctx,
-        state = %State{most_recent_frame_pts: most_recent_frame_pts}
+        state = %State{most_recent_frame_pts: most_recent_frame_pts, pads_states: pads_states}
       ) do
+    input_pads = pads_states |> MapSet.new(fn {pad, _pad_state} -> pad end)
+    :ok = Scene.validate(scene, input_pads)
```

Comment:
This validation crashes on send event, while the `VC Core` one will crash later (not immediately after sending action), due to `scene actions` enqueuing. Validating here makes it easier for VC users to spot inconsistencies with VC API in their code (this will crash immediately after API inconsistent message is sent). It may seem that the `VC Core` validation is useless in such a case, but the queue can also remove pads before sending `:update_scene` to `VC Core`, which would produce unreadable error logs. Therefore, I think we should validate it in both places.